app/connectors/AnswersConnector.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import com.google.inject.Inject
+import models.UserAnswers
+import play.api.Logger
+import play.api.libs.json.{JsObject, Json}
+import repositories.AnswersRepository
+import uk.gov.hmrc.time.DateTimeUtils
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class AnswersConnector @Inject()(val repository: AnswersRepository)(implicit ec: ExecutionContext) {
+
+  private val logger = Logger(this.getClass)
+
+  def removeByEori(eori: String): Future[Unit] = repository.remove("eori" -> eori).filter(_.ok).map(_ => (): Unit)
```

Comment:
Is `filter(_.ok)` necessary? You're mapping this to Unit, so there is no reason to use filter.

Reply:
No idea.  It's what we do in movements FE CacheRepository.